### PR TITLE
Builds are in UTC

### PR DESCRIPTION
**Pull request by hennevogel**
Make sure we do not compare across timezones.

### CODE REVIEW

**Comment by dmarcoux** [comment]
`compate`? Did you mean compare?

File: openqa-mail-notification/mail_notification.rb [hunk 1]
```diff
@@ -85,7 +85,7 @@ def send_notification(to, subject, message)
   end
 
   last_build = DateTime.parse(build['t_finished'])
-  frequency_minutes_ago = DateTime.now - (FREQUENCY / 1440.0)
+  frequency_minutes_ago = DateTime.now.new_offset(0) - (FREQUENCY / 1440.0)
```
Comment (dmarcoux) [inline openqa-mail-notification/mail_notification.rb:88]: [`0` is the default value](https://ruby-doc.org/stdlib-2.5.1/libdoc/date/rdoc/DateTime.html#method-i-new_offset), so this code could be changed to:

```suggestion
  frequency_minutes_ago = DateTime.now.new_offset - (FREQUENCY / 1440.0)
```

File: openqa-trigger/schedule-obs.rb [hunk 2]
```diff
@@ -43,7 +43,7 @@ def fetch_image(version)
 def trigger_run(version:)
   qcow2_image = fetch_image(version)
 
-  frequency_minutes_ago = DateTime.now - (FREQUENCY/1440.0)
+  frequency_minutes_ago = DateTime.now.new_offset(0) - (FREQUENCY/1440.0)
```
Comment (dmarcoux) [inline openqa-trigger/schedule-obs.rb:46]: Same comment as above.

File: openqa-mail-notification/mail_notification.rb (hunk 1 above)
Comment (hennevogel) [reply openqa-mail-notification/mail_notification.rb:88]: hm I find this less clear TBH

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**